Source/Android/jni/AndroidCommon/IDCache.h

```diff
@@ -28,4 +28,8 @@ jmethodID GetGameFileConstructor();
 jclass GetGameFileCacheClass();
 jfieldID GetGameFileCachePointer();
 
+jclass GetMapClass();
+jmethodID MapInit();
+jmethodID MapPut();
```

Comment:
```suggestion
jmethodID GetMapInit();
jmethodID GetMapPut();
```

For consistency with the other methods.